test/lib/ansible_test/_util/controller/sanity/validate-modules/validate_modules/main.py

```diff
@@ -33,8 +33,14 @@
 from contextlib import contextmanager
 from fnmatch import fnmatch
 
-from antsibull_docs_parser import dom
-from antsibull_docs_parser.parser import parse, Context
+try:
+    from antsibull_docs_parser import dom
+    from antsibull_docs_parser.parser import parse, Context
+except ImportError:
+    import traceback
+    ANTSIBULL_DOCS_PARSER_FAILURE = traceback.format_exc()
+else:
+    ANTSIBULL_DOCS_PARSER_FAILURE = None
```

Comment:
Instead of ignoring the import error, add the requirement to the unit test requirements here:

https://github.com/ansible/ansible/blob/devel/test/units/requirements.txt

Reply:
Actually, on second thought, just mock them out like we do for `voluptuous` instead:

https://github.com/ansible/ansible/blob/96f7fd5539f6124004fecef8f7110d4965686872/test/units/ansible_test/test_validate_modules.py#L12-L19

Reply:
I went ahead and took care of this change.

Reply:
I wanted to avoid that so that there are no global dependencies available for all ansible-core unit tests that are needed just for one tiny part of validate-module's code.

Now I see you found a better solution (which shouldn't have been surprising, after all validate-modules also needs `voluptuous`, and that isn't in `test/units/requirements.txt` either and it doesn't use guarded imports, so there *has* to be another way to handle them) - thanks a lot for that, that one looks best! :)